src/shell/lisa_shell

```diff
@@ -93,13 +93,27 @@ fi
 echo -ne "$LISASHELL_DEFAULT"
 }
 
+function _lisa-update-all {
+echo -ne "${LISASHELL_BLUE}"
+echo -e "*** Update LISA installation"
+
+git pull
```

Comment:
Is this not going to produce a merge commit if by chance we issue a "lisa-update all" while not being on origin/master?

Maybe we can either:
1) check before to be on origin/master (a local master branch still is not valid, since it can contain local user patches) and abort with a warning explaining that this operation is possible only while being on origin/master
2) replace this pull with:
``` shell
git fetch origin master
```
Which should safely update the origin/master head and call a sub-modules update.
Unfortunately the submodules update will not see the new origin/master... but this can be solved with a trick:
BEFORE the submodules update
- stash current modifications with a certain name (e.g. __lisa_update)
- get a reference to the current branch (CURR_BRANCH)
- checkout origin/master as a detached branch
- UPDATE submodules at this time
- go back and checkout CURR_BRANCH
- stash pop "__lisa_update"

Do you think this can work and be useful?

Otherwise I would just go for solution 1 and warn the user that she needs to be in origin/master to do an update.
It's on the user responsibilities to save the current work...